srcpkgs/hashcash/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'hashcash'
+pkgname=hashcash
+version=1.23
+revision=1
+archs="~*-musl"
+build_style=gnu-makefile
+make_build_args="LIBCRYPTO=$XBPS_CROSS_BASE/usr/lib/libcrypto.a generic-openssl docs"
```

Comment:
Sorry if I was missing anything, but what's wrong with `LIBCRYPTO=-lcrypto`?

Reply:
It wants static linking, idk why.